kbchat/test_config.example.yaml

```diff
@@ -20,7 +20,10 @@ config:
   teams:
     acme:
       # A real team that you add your alice1, alice2, and bob1 all into
-      teamname: "acme"
+      name: "acme"
+      # The channel to use
+      topicname: "mysupercoolchannel"
     alicesPlayground:
```

Comment:
this name feels like a strange deviation from the others. how about `wonka` or `cyberdyne` or `choose-your-favorite-fake-clean-one-word-company`. related, do we even need two teams? as long as one user is an admin in the team, we can do whatever setup we need before a test runs.